src/address.ts

```diff
@@ -147,9 +142,8 @@ export class Address {
   }
 
   /**
-   * Returns a random localized street name
+   * Returns a random localized street name.
```

Comment:
```suggestion
   * Generates a random localized street name.
```
(not sure about this one)

Reply:
@Shinigami92 do you know if the street names are chosen from a predetermined list?

Reply:
Street names are easily generate-able. 

- `[number]th-avenue` // 5th avenue
- `[number]th-[cardinal-direction]-street` // 5th east street
- `[first_name] street` // john street
- `[adjective] [word]street` // beautiful pinestreet
- `[adjective]-street` // nice-street

We could have a list of hardcoded street names in the background too, but that shouldn't be relevant to the user.